OXRS-SHA-StateMonitor-ESP32-FW.ino

```diff
@@ -449,6 +453,25 @@ void inputEvent(uint8_t id, uint8_t input, uint8_t type, uint8_t state)
   publishEvent(index, type, state);
 }
 
+/**
+  wrapper for oxrsInput::setType() to register config changes to/from SECURITY
+*/
+void config_input_type(uint8_t mcp, uint8_t pin, uint8_t newType)

```

Comment:
Very pedantic, sorry!, but could we keep function names consistent - e.g. `configInputType`?

Reply:
And would this be better named `setInputType()`?